### PR TITLE
feat:card comp completed

### PR DESCRIPTION
**Pull request by teja538**
please review card component
screenshots for reference:
Screenshot (237)
Screenshot (238)
Screenshot (239)
Screenshot (240)
Screenshot (241)](https://user-images.githubusercontent.com/62746249/153738872-ca7fd729-36fb-43e4-ad54-72becab7f6e0.png)
content.com/62746249/153738870-9ee06a61-8007-4b7a-8c71-b9dc3437c7ea.png)

### CODE REVIEW

File: main/dev/css/components/badge/badge.html [hunk 1]
```diff
@@ -38,7 +38,8 @@ <h2 class="pad-10">components</h2>
                         href="../alerts/alerts.html">alerts</a></li>
                     <li class="list-sn pad-10"><a class="no-decoration color-black"
                         href="../button/button.html">buttons</a></li>
-                    <li class="list-sn list-non-bullet pad-10">card</li>
+                    <li class="list-sn pad-10"><a class="no-decoration color-black"
```
Comment (raieegulhane) [inline main/dev/css/components/badge/badge.html:41]: Use proper indentation

**Review by raieegulhane — COMMENTED** [review]
Overall ooks good to go

**Review by sumitch29 — APPROVED** [review]
Looks good to me. But you can add hover effect to button inside card.

**Comment by teja538** [comment]
> Looks good to me. But you can add hover effect to button inside card.

yeah will add it. Thank you sumit

**Pull request merged**